features/s3EncryptionV2/encryptionV2.feature

```diff
@@ -1,4 +1,4 @@
-@s3EncryptionV2 @integ
+@s3EncryptionV2 @integ @requiresUniqueResources
```

Comment:
should we add this to v1 and v3?
or are we landing this first to test and then add it to the others?

Reply:
They're actually already in V1 and V3- V2 was just missing it